mods/d2k/languages/chrome/en.ftl

```diff
@@ -68,94 +68,93 @@ supportpowers-support-powers-palette =
+   .tooltipdesc = Selected units move to the target location
+   and engage any enemies they encounter en route.
 
-    Hold <(Ctrl)> while targeting to order an Assault Move
-    that attacks any units or structures encountered en route.
-
-    Left-click icon then right-click on target location.
+    Left-click the icon, then right-click on the target.
+    Hold <(Ctrl)> while targeting to issue an Assault Move,
+    that engages any units or structures encountered en route.
 
 button-command-bar-force-move =
    .tooltip = Force Move
-   .tooltipdesc = Selected units will move to the desired location
-     - Default activity for the target is suppressed
-     - Vehicles will attempt to crush enemies at the target location
-     - Deployed thumpers will undeploy and move to the target location
+   .tooltipdesc = Moves units to the target location.
+     - Overrides default behaviour.
+     - Vehicles attempt to crush enemies.
+     - Deployed thumpers undeploy and move to the target location.
 
-    Left-click icon then right-click on target.
-    Hold <(Alt)> to activate temporarily while commanding units.
+    Left-click the icon, then right-click on the target.
+    Hold <(Alt)> to temporarily activate while commanding units.
 
 button-command-bar-force-attack =
    .tooltip = Force Attack
-   .tooltipdesc = Selected units will attack the targeted unit or location
-     - Default activity for the target is suppressed
-     - Allows targeting of own or ally forces
+   .tooltipdesc = Selected units attack the targeted unit or location.
+     - Overrides default behaviour.
+     - Can target friendly or allied forces.
 
-    Left-click icon then right-click on target.
-    Hold <(Ctrl)> to activate temporarily while commanding units.
+    Left-click the icon, then right-click on the target.
+    Hold <(Ctrl)> to temporarily activate while commanding units.
 
 button-command-bar-guard =
    .tooltip = Guard
-   .tooltipdesc = Selected units will follow the targeted unit.
+   .tooltipdesc = Units follow and protect a target.
 
-    Left-click icon then right-click on target unit.
+    Left-click the icon, then right-click on the target unit.
 
 button-command-bar-deploy =
    .tooltip = Deploy
-   .tooltipdesc = Selected units will perform their default deploy activity
-     - MCVs will unpack into a Construction Yard
-     - Thumpers will start or stop attracting worms
-     - Devastators will become immobilized and explode
+   .tooltipdesc = Selected units perform their default deploy activity.
+     - MCVs unpack into a Construction Yard.
+     - Thumpers start or stop attracting worms.
+     - Devastators become immobilized and explode.
 
     Acts immediately on selected units.
 
 button-command-bar-scatter =
    .tooltip = Scatter
-   .tooltipdesc = Selected units will stop their current activity
-    and move to a nearby location.
+   .tooltipdesc = Selected units stop their current activity
+    and disperse to nearby locations.
```

Comment:
The other scatter command was changed to "and disperse to a nearby cells.". We should be consistent with our terminology.